metrics/comet/comet.py

```diff
@@ -101,8 +119,8 @@
     >>> hypothesis = ["The fire could be stopped", "Schools and kindergartens were open"]
     >>> reference = ["They were able to control the fire.", "Schools and kindergartens opened"]
     >>> results = comet_metric.compute(predictions=hypothesis, references=reference, sources=source)
-    >>> print([round(v, 2) for v in results["scores"]])
-    [0.19, 0.92]
+    >>> print([round(v, 3) for v in results["scores"]])
+    [0.839, 0.972]
```

Comment:
That seems like a very significant change (0.19->0.84) of scores between two versions. Is this expected?

Reply:
Yes it is expected. The new default model is trained with normalised scores between 0 and 1 for better interpretability.

Reply:
https://github.com/Unbabel/COMET#interpreting-scores